.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -0,0 +1,54 @@
+---
+name: Bug Report
+about: Submit a bug report to help us improve
+labels: 'bug, needs triage'
+---
+
+## Bug Report
+
+(A clear and concise description of what the bug is)
+
+## To Reproduce
+
+(Write your steps here:)
+
+1. Step 1...
+1. Step 2...
```

Comment:
2.

Reply:
Works both ways to my surprise. But I changed it.